### PR TITLE
Disable Poly on Quest 1

### PR DESCRIPTION
**Pull request by mikeskydev**
A cleaner fix for disabling Poly on Quest 1 that doesn't block Quest 2.

Adds an extra popup to the Sketchbook and Poly panels notifying the user that the feature isn't available.

com Icosa OpenBrush-20210406-223532

### CODE REVIEW

**Comment by mikeage** [comment]
LGTM; a much better user experience than the current method!

[A reminder to anyone interested in testing the builds here; poly support requires the Secrets.asset file, which is not available in the PR builds, so don't try.]

**Comment by mikeskydev** [comment]
Here's a working APK to test with. I don't have a Quest 2 so I can't verify that they don't get locked out! https://drive.google.com/file/d/1IDS1zDX2XcAMYbCux959QOHKXPVbDEYo/view?usp=sharing

**Comment by mikeage** [comment]
I'll check tonight (missed this last night, and I'm already at work now ;-) )

**Comment by mikeage** [comment]
Looks good on Q2!

**Comment by mikeage** [comment]
@mikeskydev , this seems to have broken the OS X Monoscopic build; it doesn't get past the loading screen. I don't know if it's something related to the new files and the cache or something else; I'll try rebuilding cleanly on my local machine, but I'll be offline until Sat night. In the meantime, any ideas what might have happened?

File: Assets/Scripts/GUI/PolyPanel.cs [hunk 1]
```diff
@@ -64,6 +65,7 @@ public class PolyPanel : ModalPanel {
     m_NoLikesMessage.SetActive(false);
     m_NotLoggedInMessage.SetActive(false);
     m_OutOfDateMessage.SetActive(false);
+    m_NotSupportedMessage.SetActive(false);
```
Comment (mikeskydev) [inline Assets/Scripts/GUI/PolyPanel.cs:68]: The actual code changes are gated behind platform defines, I can only wonder if it's giving a NRE when trying to disable here.

Comment (mikeskydev) [reply Assets/Scripts/GUI/PolyPanel.cs:68]: This should have broken other desktop builds if so, how are the others?

Comment (mikeage) [reply Assets/Scripts/GUI/PolyPanel.cs:68]: Is there a way I can see logs?

Comment (mikeskydev) [reply Assets/Scripts/GUI/PolyPanel.cs:68]: not without doing a dev build i think - i'll give it a test tonight

Comment (mikeage) [reply Assets/Scripts/GUI/PolyPanel.cs:68]: No idea, OS X is my only platform (and Q2). I'll try a build too, but won't be able to run until Sat. night.

If it helps, it hangs and CPU shoots way up, but there's no crash or exit

Comment (mikeage) [reply Assets/Scripts/GUI/PolyPanel.cs:68]: Ok, I did manage to get a build in before I sign off. This one broke it; reverting it against the tip fixes it. Thanks

Comment (mikeskydev) [reply Assets/Scripts/GUI/PolyPanel.cs:68]: Okay, easy enough fix to make it nullable! Thanks for checking

Comment (billyquith) [reply Assets/Scripts/GUI/PolyPanel.cs:68]: Just looking into this and found the same issue.

Comment (mikeage) [reply Assets/Scripts/GUI/PolyPanel.cs:68]: I can confirm that #91 fixes it

**Pull request merged**